pkg/kwokctl/vars/vars.go

```diff
@@ -171,11 +171,28 @@ var (
 		return ""
 	}()
 
+	// Mode is several default parameter templates for clusters
+	Mode = getEnv("KWOK_MODE", "")
+
+	// ModeTiny efforts to reduce cluster configuration requirements
+	// Disables all alpha feature by default, as well as Bata feature that are not eventually GA
+	ModeTiny = "tiny"
```

Comment:
"tiny" doesn't read well, can we change to another name? like `StableFeatureFateAndAPI`?

Reply:
Updated